### PR TITLE
Ping pong game coderchirag

### PR DESCRIPTION
**Pull request by CoderChirag**

### CODE REVIEW

**Comment by CoderChirag** [comment]
Now check the game, I have created a new branch. And if you find it good then merge please

**Comment by Harshil-Jani** [comment]
@CoderChirag 
Add Readme file in your Project Folder and include following details.
- Project Name
- Short Description
- .gif of working project.
- Author name 

And we are good to go ! :star2:

**Comment by CoderChirag** [comment]
Added the readme file also. Now check and merge please

File: ping-pong game/index.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    <link rel="stylesheet" href="style.css">
+    <title>Ping Pong</title>
+</head>
+<body>
+    <div id="upper" class="bars"></div>
+    <div id="lower" class="bars"></div>
+    <div id="ball"></div>
+    <script src="index.js"></script>
+</body>
+</html>
```
Comment (soumik2012) [inline ping-pong game/index.html:15]: Add a new line at the EOF.

File: ping-pong game/index.js [hunk 2]
```diff
@@ -0,0 +1,317 @@
...
+    }
+    else if (e.key=="ArrowRight" && isStarted && barX<maxBarX){
+        barX++;
+        setBar();
+    }
+    else if (e.key=="ArrowLeft" && isStarted && barX>minBarX){
+        barX--;
+        setBar();
+    }
+});
```
Comment (soumik2012) [inline ping-pong game/index.js:317]: Add a new line at the EOF.

File: ping-pong game/style.css [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+}
+#ball{
+    height: 20px;
+    width: 20px;
+    border-radius: 50%;
+    background-color: red;
+    position: fixed;
+    left: 47.5%;
+    top: 94.5%;
+}
```
Comment (soumik2012) [inline ping-pong game/style.css:23]: Add a new line at the EOF.

**Review by soumik2012 — CHANGES_REQUESTED** [review]
The response time of the ball is very slow Fixed it ...

**Comment by CoderChirag** [comment]
@Harshil-Jani bro, Now if this project is okk, then please merge it

**Comment by soumik2012** [comment]
@CoderChirag where are your files???

**Comment by CoderChirag** [comment]
@soumik2012  see the **Files changed** tab above. There you would find all the files.

**Comment by CoderChirag** [comment]
@Harshil-Jani bro when would you merge it bro? Is there any mistake left now?

File: ping-pong game/index.html (hunk 1 above)
Comment (Harshil-Jani) [inline ping-pong game/index.html:15]: Create a new line at end of file

File: ping-pong game/index.js (hunk 2 above)
Comment (Harshil-Jani) [inline ping-pong game/index.js:317]: Create a new line at end of file

File: ping-pong game/style.css (hunk 3 above)
Comment (Harshil-Jani) [inline ping-pong game/style.css:23]: Create a new line at end of file

**Comment by Harshil-Jani** [comment]
@CoderChirag Just create new lines wherever asked and then we can merge it !

**Comment by CoderChirag** [comment]
@Harshil-Jani  bro done. Now merge it please

**Review by Harshil-Jani — APPROVED** [review]

**Pull request merged**